packages/modules/link-modules/src/services/dynamic-service-class.ts

```diff
@@ -5,7 +5,12 @@ export function getModuleService(
   joinerConfig: ModuleJoinerConfig
 ): Constructor<ILinkModule> {
   const joinerConfig_ = JSON.parse(JSON.stringify(joinerConfig))
+  const databaseConfig = joinerConfig_.databaseConfig
+
   delete joinerConfig_.databaseConfig
+
+  joinerConfig_.extraFields = Object.keys(databaseConfig.extraFields || {})
```

Comment:
I think we can rename this to extraDataFields to do not be mixed up to the one inside databaseConfig, andI  believe if there is something already defined on this field we must validate if all of them are defined inside databaseConfig, or throw.